### DISCUSSION TITLE
Property to associate a markdown file to a PDF file - What is it for

### DISCUSSION THREAD

**Discussion by codingThome** [RyotaUshio/obsidian-pdf-plus#144] [Q&A] [answered]
I have no idea how to use the PDF-Property (or Dataview field) respectively I cannot get this to work.
What should this do???
Thanks for this great plugin!!!

**Comment by N3C2L** [comment, marked as answer]
If you don't know what properties are check out the help file from the obsidian website "Properties":
https://help.obsidian.md/Editing+and+formatting/Properties

Inside the PDF++ settings you have "PDF" as a property as a default value.
You can open your properties sidebar inside obsidian and add "PDF" with the value [[myfavoritebook.pdf]]

Your annotation file is now associtiated with this .pdf file. This is important  if you want to use other properties inside this annotation file when you copy links from the PDF to the annotation file. 

If you use Zotero, you can look at this interesting property list from @RyotaUshio :

https://gist.github.com/RyotaUshio/ca6eae6b830e11a71b8fa101d5f05d67

He then uses for example the citet property from this list and adds it to the display text format pictured in this link:

https://github.com/RyotaUshio/obsidian-pdf-plus/wiki/Example-workflow:-Zotero-integration

**Reply by anhnguyenle193** [reply]
Hello,
Could you please join a 'obsidian://pdf-plus?setting=' link to the settings of the default value for this property?
I cannot find it among the numerous other PDF++ settings.
Thank you.
